.github/CHANGELOG.md

```diff
@@ -74,6 +74,7 @@
 1. [FUEL] Lowered starting fuel on C/D spawn, will only load last saved fuel on C/D spawn, center tank refuel now happens simultaneous with wing refuel - @Maximilian-Reuter
 1. [EFB/SIMBRIEF] Option to import SimBrief Fuel & Payload when SimBrief Data is imported - @Fragtality (Fragtality) + @Maximilian-Reuter
 1. [FLIGHTMODEL] Fixes some crosswind issues - @donstim (donbikes)
+1. [LIGHTS] Movement of landing lights now requires power and position is output into LVAR -  @Maximilian-Reuter
 1. [PFD] Add true bearing flags & refactor init - @alepouna, @tracernz
```

Comment:
Don't think we're doing A380X changelogs yet? In any case the A32NX already has this so the changelog is a bit misleading.

Reply:
Oh ok.